packages/rx_bloc_cli/mason_templates/bricks/feature_deeplink/__brick__/lib/feature_deep_link_list/views/deep_link_list_page.dart

```diff
@@ -25,7 +25,7 @@ class DeepLinkListPage extends StatelessWidget {
   Widget build(BuildContext context) => Scaffold(
         appBar: customAppBar(
           context,
-          title: context.l10n.featureDeepLink.deepLinkFlowPageTitle,
+          title: context.l10n.deepLinkFlowPageTitle,
```

Comment:
Why the `featureDeepLink` namespace is not used anymore ?

Reply:
restored